plugins/keyboard-navigation/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "blockly-plugin-keyboard-navigation",
+  "version": "0.20201029.0",
+  "description": "A Blockly plugin.",
+  "scripts": {
+    "audit:fix": "blockly-scripts auditFix",
+    "build": "blockly-scripts build",
+    "clean": "blockly-scripts clean",
+    "lint": "blockly-scripts lint",
+    "prepublishOnly": "npm run clean && npm run build",
+    "start": "blockly-scripts start",
+    "test": "blockly-scripts test"
+  },
+  "main": "./dist/index.js",
+  "module": "./src/index.js",
+  "unpkg": "./dist/index.js",
+  "author": "Blockly Team",
+  "private": true,
+  "keywords": [
+    "blockly",
+    "blockly-plugin",
+    "keyboard-navigation"
+  ],
+  "homepage": "https://github.com/alschmiedt/blockly-samples/tree/master/plugins/keyboard-navigation#readme",
```

Comment:
I know you're not publishing yet but you might want to go ahead and update this (and others) to point to google instead of your fork so you don't forget later.

Reply:
done